sky/serve/autoscalers.py

```diff
@@ -78,10 +80,10 @@ class RequestRateAutoscaler(Autoscaler):
 
     def __init__(self,
                  *args,
-                 query_interval: int = 10,
-                 upper_threshold: int = 10,
-                 lower_threshold: int = 2,
                  min_nodes: int = 1,
+                 max_nodes: Optional[int] = 10,
+                 upper_threshold: Optional[int] = 1,
+                 lower_threshold: Optional[int] = 0,
```

Comment:
should we set the default of three arguments to be `None` given the default policy will be fixed #replica.